README.md

```diff
@@ -12,7 +15,7 @@ many other python packages along with it. To avoid any side effects for your def
 python installation, we recommend installation in a
 [virtual environment](https://virtualenv.pypa.io/en/stable/).
 
-Some code of `pylexibank` relies on LingPy functionality which is not yet released, thus,
+Some code of `pylexibank` relies on [LingPy](http://lingpy.org/) functionality which is not yet released, thus,
```

Comment:
What are we actually relying on here? Can we list what this is, so we know when lingpy has released it?